frontend/src/pages/projects/screens/spawner/imageSelector/ImageStreamSelector.tsx

```diff
@@ -47,7 +176,7 @@ const ImageStreamSelector: React.FC<ImageStreamSelectorProps> = ({
               isCompatibleWithIdentifier(identifier, imageStream),
             ) && (
               <Label color="blue">
-                Compatible with {isHardwareProfilesAvailable ? 'hardware profile' : 'accelerator'}
+                Compatible with {isHardwareProfilesAvailable ? ' hardware profile' : ' accelerator'}
```

Comment:
was the space after 'with' getting ignored here?

Reply:
No 😢 
<img width="674" alt="Screenshot 2025-04-04 at 3 56 51 PM" src="https://github.com/user-attachments/assets/d294c17c-09c6-4a66-8532-ff30dae3de5e" />